Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -40,7 +40,7 @@ public Player Player
         }
         public PowerUpsContainer PowerUpsContainer
         {
-            get => GameState.PowerUpsContainer;
+            get => GameState?.PowerUpsContainer;
             set => GameState.PowerUpsContainer = value;
```

Comment:
? `GameState` only potentially null on the getter? Why?

Reply:
Had a weird bug with comboboxes.